documentation/modules/configuring/proc-config-mirrormaker2-replication.adoc

```diff
@@ -220,9 +227,8 @@ To configure topic offset synchronization, this property must also be set for th
 <32> Optional setting to synchronize consumer group offsets, which is useful for recovery in an active/passive configuration. Synchronization is not enabled by default.
 <33> If the synchronization of consumer group offsets is enabled, you can adjust the frequency of the synchronization.
 <34> Adjusts the frequency of checks for offset tracking. If you change the frequency of offset synchronization, you might also need to adjust the frequency of these checks.
-<35> Topic replication from the source cluster xref:type-KafkaMirrorMaker2MirrorSpec-reference[defined as regular expression patterns]. Here we request all topics.
-<36> Consumer group replication from the source cluster xref:type-KafkaMirrorMaker2MirrorSpec-reference[defined as regular expression patterns]. Here we request three consumer groups by name.
-You can use comma-separated lists.
+<35> Topic replication from the source cluster xref:type-KafkaMirrorMaker2MirrorSpec-reference[defined as a comma-separated list or regular expression pattern]. Used by the source connector to replicate specified topics. Here we request three topics by name.
+<36> Consumer group replication from the source cluster xref:type-KafkaMirrorMaker2MirrorSpec-reference[defined as a comma-separated list or regular expression pattern]. Used by the checkpoint connector to replicate specified consumer groups. Here we request three consumer groups by name.
```

Comment:
From the discussion on the issue, that does not seem to be completely clear. The examples David shared seem to suggest otherwise. So I think we need some more investigation here in terms of what is used where.

Reply:
I've updated to include checkpoint connector in callout 35